activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+*   Add `ActiveRecord::Base.suppress` to allow save callbacks to be suppressed
```

Comment:
It prevents saves, not save callbacks. It is useful to prevent undesirable saves happening in callbacks.